src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -667,19 +667,37 @@ private void LoadNodeViewCustomizations()
         {
             nodeViewCustomizationLibrary.Add(new CoreNodeViewCustomizations());
             foreach (var assem in dynamoViewModel.Model.Loader.LoadedAssemblies)
+            {
                 nodeViewCustomizationLibrary.Add(new AssemblyNodeViewCustomizations(assem));
+               
+            }
+            //add customizations from each NodeViewLibrary in all loaded packages.
+            foreach(var assem in dynamoViewModel.Model.GetPackageManagerExtension().PackageLoader.LocalPackages.SelectMany(x => x.NodeViewLibraries))
```

Comment:
How will the node view customization libraries get added to `LocalPackages `in the first place?

Reply:
The binaries are loaded when the package is loaded, and are set as NodeViewLibraries here:
`IsNodeViewOnlyLibrary = IsNodeViewLibrary(nodeViewLibraries, assem.GetName(), warnings)`
https://github.com/DynamoDS/Dynamo/pull/11891/files#diff-eb9c2a35d407323409fa2fc260e277955b068bc335e05165dc6156bf6284f9a7R397